<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSearch</class>
 <widget class="QDialog" name="DialogSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="title">
    <string>Метод поиска</string>
   </property>
   <widget class="QRadioButton" name="radioButton_fullSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>полное соотвествие</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_partSearch">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>неполное соответствие</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_wildcard">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>по шаблону</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="title">
    <string>Режим поиска</string>
   </property>
   <widget class="QRadioButton" name="radioButton_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>по наименованию</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_code">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>по коду</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>461</width>
     <height>401</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="title">
    <string>Критерии</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="widget_class2" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>41</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Класс:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_class">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>2</y>
       <width>256</width>
       <height>131</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_class1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_class2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_class1" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox_codeInGroup">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Не важно</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetGroup" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>41</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Группа:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_group">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>Не важно</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Код в группе:</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetClass" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>41</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Класс</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_class">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_type" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>41</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_type">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>2</y>
       <width>256</width>
       <height>61</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Уровень:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_level">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Текст:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_find">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_close">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_class1</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_class1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_class2</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_class2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pButton_close</sender>
   <signal>clicked()</signal>
   <receiver>DialogSearch</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
